### ISSUE TITLE
Could not load ROM

### ISSUE THREAD

**Issue by OmegaAa75** [rafaelvcaetano/melonDS-android#62]
* **Device model:** moto g(7)
* **Android version:** Android API 29

**Problem:**  
Insert a small description of the problem.

**Stack trace:**  
```
me.magnum.melonds.ui.emulator.EmulatorActivity$f: Failed to load ROM: NDS_FAILED
	at me.magnum.melonds.ui.emulator.j$c$a$a.a(Unknown Source:205)
	at f.a.b0.e.e.a.m(Unknown Source:10)
	at f.a.s.a(Unknown Source:14)
	at f.a.b0.e.b.e$a.d(Unknown Source:26)
	at f.a.b0.e.b.l$a.d(Unknown Source:2)
	at f.a.b0.e.b.j$a.run(Unknown Source:21)
	at f.a.r$a.run(Unknown Source:9)
	at f.a.b0.g.j.run(Unknown Source:13)
	at f.a.b0.g.j.call(Unknown Source:0)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:301)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:919)

```

**Comment by rafaelvcaetano** [comment]
Could you check if the Storage permission is granted? You can see this from the app's info screen by following these steps:

1. Open the device's Settings
2. Go to Apps
3. Find and select "melonDS"
4. Click on Permissions
5. Check if the Storage permission is enabled

If the permission is enabled, then it's possible that you have an invalid BIOS file. If the permission is disabled, try enabling it and try again. If it was disabled, let me know since that shouldn't happen.

Could you also let me know where you placed your BIOS files? Was it in the internal storage? Or in an SD card?

**Comment by shabbirmahmud** [comment]
Can you use the latest beta [v1.3.0](https://github.com/rafaelvcaetano/melonDS-android/releases/tag/beta-1.3.0) and open a new issue? The latest version does not fix the problem but includes additional info that will help to find the cause.

**Comment by rafaelvcaetano** [comment]
If your problem is not yet fixed, try the latest [Beta 1.4.0](https://github.com/rafaelvcaetano/melonDS-android/releases/latest) release. If the problem is not fixed, please open a new issue.
